GlimpseImputationPipeline/Glimpse2SplitReference.wdl

```diff
@@ -0,0 +1,166 @@
+version 1.0
+
+workflow Glimpse2SplitReference {
+    input {
+        # There are two variables required to set define the contigs. The first one, contig_regions, defines the coordinates for the separate chunks,
+        # usually this would be an array of ["chr1", "chr2", ..., "chr22", "chrX:1-2781479", "chrX:2781480-155701382", "chrX:155701383-156030895"].
+        # Notice the split of chrX into PAR1, non-PAR, PAR2. When constructing the reference panel chunks, however, we need to determine the appropriate
+        # reference panel file names. We could do that by parsing the substring before the colon (which WDL doesn't support), or we can just pass
+        # another array contig_names_in_reference_panel = ["chr1", "chr2", ..., "chr22", "chrX", "chrX", "chrX"]. Note that contig_regions and
+        # contig_names_in_reference_panel must have the same length.
+        Array[String] contig_regions
+        Array[String] contig_names_in_reference_panel
+        Array[String] contig_names_in_genetic_maps
```

Comment:
Yes, I had to add this because of the par regions. In the genetic maps, the par regions are called e.g. "chrX_par1" but the sites for that region are still in the complete "chrX" VCF